app/assets/styles/bulletin.css

```diff
@@ -46,123 +41,111 @@ hr.image {
 	font-size: 0.8em;
 }
 
-.bupage-hero--title {
+.wrapper {
+	padding: 2em;
+	max-width: 87vw;
+	margin-left: auto;
+	margin-right: auto;
+	margin-top: 4em;
+}
+
+.wrapper__title {
 	color: #000099;
 	font-weight: 300;
 	font-size: 3rem;
 	margin-bottom: 0;
 }
 
-.bupage-hero--subtitle {
+.wrapper__subtitle {
 	color: #808080; 
 	font-weight: 300;
 	font-size: 2rem;
 	margin-top: 0;
 	margin-bottom: 1.2em;
 }
 
-.bupage {
-	padding: 2em;
-	max-width: 87vw;
-	margin-left: auto;
-	margin-right: auto;
-	margin-top: 4em;
-}
-
-.buarchive {
+.article-content {
 	display: grid;
 	grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
 	grid-gap: 1em;
 	grid-auto-flow: dense;
 }
 
-.buarticle {
+.content-section {
 	padding: 1em;
 	background: #fff;
 	box-shadow: 
 		0 5px 10px rgba(0, 0, 0, 0.1),
 		0 20px 20px rgba(0, 0, 0, 0.05);
 }
 
-.buarticle-hero--large {
+.content-section__hero_large {
 	color: #00004d;
 	font-size: 2em;
 	font-weight: 300;
 }
 
-.buarticle-main--title {
+.content-section__headline {
 	font-size: 300;
 	color: #00004d;
 }
 
-.buarticle-heading {
-	font-weight: 400;
-	font-size: 1.35em;
-	color: black;
-}
-
-.buarticle-blue {
+.content-section_blue {
 	color: white;
 }
```

Comment:
Do we need this class? If we can remove it, let's remove it. If we can't, it should be named to reflect the color.